src/backend/api/reviews.js

```diff
@@ -0,0 +1,96 @@
+const { query } = require("express");
+const express = require("express");
+const { max } = require("../database");
+const router = express.Router();
+const knex = require("../database");
+
+router.get('/', async (req, res) => {
+    try {
+        const reviewList = await knex('review').select('*')
+        if (!reviewList.length) {
+            res.status(404).json("There are not available any reviews")
+        } else {
+            res.json(reviewList);
+        }
+    } catch (error) {
+        throw res.status(404).json({ error: "There is error in Page try again" });
+    }
+});
+
+router.get('/:id', async (req, res) => {
+    try {
+        const id = req.params.id
+        const reviewList = await knex.from('review').select('*').where('meal_id', id);
```

Comment:
You are looking up the review by the meal ID. So `GET /api/reviews/14` would give me the reviews for meal with ID 14. I think you instead want to get the review by ID, not meal ID.

Reply:
yaah.. you are right...